pom.xml

```diff
@@ -2082,23 +2082,6 @@
           <artifactId>maven-deploy-plugin</artifactId>
           <version>2.8.2</version>
         </plugin>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-dependency-plugin</artifactId>
-          <executions>
-            <execution>
-              <id>default-cli</id>
-              <goals>
-                 <goal>build-classpath</goal>
-              </goals>
-              <configuration>
```

Comment:
(I'll open a bug also since this now becomes more complicated than I initially thought.)